sorc/link_workflow.sh

```diff
@@ -172,7 +172,7 @@ if [[ -d "${script_dir}/gdas.cd" ]]; then
   cd "${top_dir}/fix" || exit 1
     [[ ! -d gdas ]] && mkdir -p gdas
     cd gdas || exit 1
-    for gdas_sub in bump crtm fv3jedi gsibec; do
```

Comment:
Other UFS-DA applications use `bump`.  While EIB currently does not stage these `bump` files, I'm guessing that you eventually want to stage `bump` files in a central location as opposed to each clone maintaining a local copy.  If this is true, do you want to retain the `bump` directory (now empty) as a visual placeholder?

Reply:
I don't mind keeping an empty directory around.

Reply:
Thank you @aerorahul

Reply:
i.e. it will be a dead link.

Reply:
I didn't run `link_workflow.sh` to see what happens.  Dead links aren't good.   Seems we should forget my comment and move on.